consensus/types/src/chain_spec.rs

```diff
@@ -1302,6 +1302,30 @@ impl ChainSpec {
             domain_bls_to_execution_change: 10,
         }
     }
+
+    /// The current sync committee is considered misaligned if a fork is scheduled inside it.
+    /// That is, if `fork_epoch` is not divisible by `epochs_per_sync_committee_period` and the current
+    /// sync committee period is equal to the `fork_epoch` sync committee period.
+    pub fn is_current_sync_committee_period_misaligned<E: EthSpec>(
+        &self,
+        current_slot: Slot,
+    ) -> bool {
+        let is_fork_boundary_inside_sync_committee_period = |fork_epoch: Epoch| {
```

Comment:
There's an existing check below, but it only log a warning if it's misaligned. Perhaps we can remove that one given that it would be a bit more obvious now if the fork schedule is misaligned?

https://github.com/sigp/lighthouse/blob/a1b7d616b47604ec0cd1afb5543e03e68b629f96/beacon_node/src/lib.rs#L190